ext/src/rubydoop/InstanceContainer.java

```diff
@@ -49,10 +46,8 @@ public static InstanceContainer lookupClass(Configuration conf, String rubyClass
     }
 
     private static Object lookupClassInternal(ScriptingContainer runtime, Configuration conf, String rubyClassProperty) {
-        String jobConfigScript = getRequired(conf, JOB_SETUP_SCRIPT_KEY);
         String rubyClassName = getRequired(conf, rubyClassProperty);
         try {
-            runtime.callMethod(null, "require", jobConfigScript);
```

Comment:
As I mentioned in the commit message; I'm not sure this is a safe thing to remove, but it seems to be working without it.